source/2023.tex

```diff
@@ -0,0 +1,100 @@
+
+\subsection{Trustees}
+\begin{tabular}{l l}
+	Ann Barr       & Gail Ollis    \\
+	Owen Campbell  & Vincent Prytherch        \\
+	Kristian Glass & Rebecca Smith \\
+\end{tabular}
+
+\section{Objectives and Activities}
+The objectives of the UKPA are:
+\begin{enumerate}
+	\item To advance education for the public benefit in the use and understanding of the Python programming language within the UK by:
+	      \begin{enumerate}
+		      \item Organising, presenting and promoting Python related conferences, meetings and events at venues within the UK,
+		      \item Supporting and promoting Python related projects and events organised by UKPA members, and,
+		      \item Supporting UKPA members wishing to attend Python related conferences, meetings and events.
+	      \end{enumerate}
+	\item To advance education for the public benefit in the use and understanding of the Python programming language worldwide by supporting and promoting overseas Python related conferences, meetings and events.
+\end{enumerate}
+
+The primary activity of the UKPA is to organise and run the annual PyCon UK conference.
+
+\section{Structure, Governance and Management}
+
+\subsection{Governing Document}
+The UKPA's governing document is its constitution, the current version of which is available at https://github.com/PyconUK/ukpa-constitution/releases/latest.
+
+\subsection{UKPA Members}
+The UKPA membership is the electoral body to which the trustees are accountable.
+There are two classes of membership:
+
+\begin{enumerate}
+	\item Members: have the full rights and obligations described in the UKPA constitution, including the right to vote on members decisions.
+	\item Associate Members: non-voting class of membership as described in the UKPA constitution.
+\end{enumerate}
+
+\subsection{Trustee Appointment}
+At the AGM in 2022, Owen Campbell and Kristian Glass retired as a trustees in accordance with section 13.1.1 of the UKPA's constitution.
+Owen Campbell and Kristian Glass were elected as trustees for a further term in accordance with section 13.1.4 of the UKPA's constitution.
+
+At a Trustees Meeting on 19 November 2022, Rebecca (Becky) Smith was appointed by the trustees as a trustee of the UKPA in accordance with clause 13.1.4 of the constitution
+At a Trustees Meeting on 04 March 2023, the office held by Tania Allard was vacated in accordance with clause 15.1.2 of the constitution.
+\subsection{Risk and Internal Control}
+The trustees have identified the major risks to which they believe the UKPA is exposed. These are:
+
+\subsubsection{Reduction or Loss of Members}
+If there were a reduction in membership of the UKPA then there would have to be a contraction, consolidation or closure of its activities.
+
+\subsubsection{Injury to Members, Conference Attendees or Trustees}
+The UKPA requires insurance to cover this potential risk and the trustees intend to put that insurance in place once the UKPA has sufficient funds to cover the cost of the premium.
```

Comment:
```suggestion
```

Reply:
I don't see why the UKPA would insure conference attendees; that's something for UKPATL

Reply:
Or is that a suggested deletion rendering oddly on mobile

Reply:
Yes, it's a suggested deletion. I'll accept it